packages/common/src/jsonApiClient/utilities/splitRelationships.js

```diff
@@ -3,26 +3,42 @@ const {
   getResourceRelationshipKeysToIncludeInBodyMap,
 } = require('../../schemaInterface')
 
+const shouldModifyRelationship = require('./shouldModifyRelationship')
+
 const resourceRelationshipKeysToIncludeInBodyMap = getResourceRelationshipKeysToIncludeInBodyMap()
 
 const dep = new Dependor({
   resourceRelationshipKeysToIncludeInBodyMap,
+  shouldModifyRelationship,
 })
 
-function splitRelationships({ itemResourceType, relationships } = {}) {
+function splitRelationships(
+  { itemResourceType, relationships, relationshipsToModify, resourcePath } = {}
+) {
   const res = {
     relationshipsToAssociateSeparately: {},
     relationshipsToIncludeInRequest: {},
+    relationshipsToNotUpdate: [],
   }
 
   const relationshipKeysToIncludeInBody =
     dep.resourceRelationshipKeysToIncludeInBodyMap[itemResourceType] || []
 
-  Object.keys(relationships).forEach(key => {
-    if (relationshipKeysToIncludeInBody.includes(key)) {
-      res.relationshipsToIncludeInRequest[key] = relationships[key]
+  Object.keys(relationships).forEach(relationKey => {
+    if (
+      !shouldModifyRelationship({
+        relationKey,
+        relationshipsToModify,
+        resourcePath,
+      })
+    ) {
+      res.relationshipsToNotUpdate.push(relationKey)
```

Comment:
naming thing, usually `modify` is used, but here it's `ToNotUpdate`. maybe `relationshipsToModify` and `relationshipsToNotUpdate` are not exact opposites, but if they are `ToNoModify` would be more consistent (and not make the reader wonder if it's a different thing)